### PR TITLE
Update install.bat

### PR DESCRIPTION
**Pull request by simon-visser**
Added -p to mkdir so that make directory even if exists

### CODE REVIEW

File: install.bat [hunk 1]
```diff
@@ -1,4 +1,4 @@
-mkdir C:\securityTrainingScript
+mkdir -p C:\securityTrainingScript
```
Comment (KevinHelfrich) [inline install.bat]: Can you point me too something showing this works on windows.

-p seems to be a linux only flag from what I can find on google, and my own testing.

The rest of the code in the repo is windows specific, so any fix has to work on windows.

Comment (KevinHelfrich) [reply install.bat]: Maybe try something along the lines of:

https://stackoverflow.com/questions/4165387/create-folder-with-batch-but-only-if-it-doesnt-already-exist

**Comment by simon-visser** [comment]
Quite right, I've changed the commit to reflect your suggestions. That SO answer is perfect, and solution is tested.

**Review by KevinHelfrich — APPROVED** [review]

**Comment by KevinHelfrich** [comment]
Thanks. Verified it as well. Appears to work. Merged it.

**Pull request merged**